offerzen/global/employers/employers@1.0.js

```diff
@@ -0,0 +1,361 @@
+      var searchParams = new URLSearchParams(window.location.search)
+      // Set skills_hiring_for
+      var hiringSkills = []
+      $('.text-span-skills').each(function () {
+        hiringSkills.push($(this).text())
+      })
+      var uniqSkills = new Set(hiringSkills)
+      const formData = new FormData(form[0])
+      const formProperties = Object.fromEntries(formData.entries())
+      const role_types = getRoleTypes(formData, formProperties)
+
+      /*if (role_types.length < 1) {
+          $('.role-type-error-container').html(
+            '<ul class="parsley-errors-list filled js-missing-fields"><li class="parsley-required">This field is required</li></ul>'
+          )
+        } else {
+          $('.role-type-error-container').html('')
+        }*/
+      if (form.parsley().validate() /* && role_types.length > 0 */) {
+        $('.js-missing-fields').hide()
+        tracking()
+        $.ajax({
+          type: 'POST',
+          url: '/zadev/company/form_leads',
+          data: JSON.stringify(
+            Object.assign({}, formProperties, {
+              workplace_policy: role_types.join(','),
+              skills_hiring_for: Array.from(uniqSkills).join(', '),
+              referrer: location.href,
+              report_source: searchParams.get('report_source'),
+              'g-recaptcha-response-data': {
+                webflow: token,
+              },
+            })
+          ),
+          contentType: 'application/json',
+          headers: {
+            Accept: 'application/json',
+          },
+          success: function (data) {
+            if (data.user_id) {
+              window.location.href = data.redirect_url
+            }
+          },
+        })
+      } else {
+        $('.js-missing-fields').show()
+        form.find('input[type=submit]').attr('disabled', false)
+        form.find('input[type=submit]').attr('value', initialButtonValue)
+      }
+    })
+  }
+  ;(function updateSubscribeToHiringInsightsField() {
+    subscribeToCompanyNewsletter = $('#subscribe_to_company_newsletter')
+    subscribeToCompanyNewsletter.on('change', function () {
+      document.getElementById('subscribe_to_hiring_insights').value = this.value
+    })
+  })()
+
+  //Intl-tel-input
+  var input = document.querySelector('#phone-number'),
+    dialCode = document.querySelector('#dial_code'),
+    contact_phone = document.querySelector('#contact_phone')
+
+  var iti = intlTelInput(input, {
+    initialCountry: 'za',
+    placeholderNumberType: 'FIXED_LINE',
+    autoPlaceholder: 'polite',
+    preferredCountries: ['za', 'nl', 'de'],
+    autoHideDialCode: false,
+    separateDialCode: true,
+    dropdownContainer: document.getElementById('js-phone-dropdown'),
+  })
+
+  var updateContactPhoneValue = function (event) {
+    dialCode.value = '+' + iti.getSelectedCountryData().dialCode
+    contact_phone.value =
+      dialCode.value +
+      input.value.replace(/[ \-\(\)]/g, '').replace(/\+([0-9]{2})|([0])/g, '')
```

Comment:
This has the bug in it that's causing [GPS-951](https://offerzen.atlassian.net/browse/GPS-951)